tests/cases/conformance/jsdoc/callbackTagNestedParameter.ts

```diff
@@ -0,0 +1,21 @@
+// @noEmit: true
```

Comment:
Use `@emitDeclarationOnly` instead so we can see the .d.ts emit

```suggestion
// @emitDeclarationOnly: true
```

Reply:
It is a long time ago, but I think I was having some problems with this. Are you sure, this works? In other words, have you run this test with the suggested change, or should I do so?

Reply:
Yes, I’m sure this works in general. I didn’t try it with this PR specifically, but I don’t see why it wouldn’t. This is kind of the standard way we’ve been baselining JS file tests.